autotest/gdrivers/gpkg.py

```diff
@@ -3374,6 +3374,69 @@ def test_gpkg_float32_png_negative_values():
 
     gdal.Unlink('/vsimem/tmp.gpkg')
 
+###############################################################################
+# Test creating a dataset with one of 'undefined' SRS
+
+def test_gpkg_write_srs_undefined_geographic():
```

Comment:
To avoid the test suite to grow too much, as the underlying code that is tested in the raster & vector side is the same, I would drop the tests in that file and just keep the ones of ogr_gpkg.py